Code/GraphMol/MolDraw2D/MolDraw2DCairo.cpp

```diff
@@ -56,6 +56,40 @@ void MolDraw2DCairo::drawLine(const Point2D &cds1, const Point2D &cds2) {
   cairo_stroke(dp_cr);
 }
 
+void MolDraw2DCairo::drawWavyLine(const Point2D &cds1, const Point2D &cds2,
+                                  const DrawColour &col1,
+                                  const DrawColour &col2,
+                                  unsigned int nSegments, double vertOffset) {
+  PRECONDITION(dp_cr, "no draw context");
+
+  if (nSegments % 2)
+    ++nSegments;  // we're going to assume an even number of segments
+
+  Point2D perp = calcPerpendicular(cds1, cds2);
+  Point2D delta = (cds2 - cds1);
+  perp *= vertOffset;
+  delta /= nSegments;
```

Comment:
The code is happy to draw bonds of zero length. Should be able to draw zero-length wavy lines too. (These are obviously not useful, but neither are they really errors. Checking for them everywhere in order to generate warnings would add a fair amount of overhead for warnings that are very likely to be ignored)